$NetBSD: patch-aa,v 1.3 2023/02/28 07:14:43 wiz Exp $

Patches remove the reference to bdftruncate (never used) and mkfontscale
(bitmap fonts only). Perl is not needed any longer as site effect and
should have never been checked for in first place.

--- Makefile.am.orig	2023-02-28 00:55:38.000000000 +0000
+++ Makefile.am
@@ -208,10 +208,6 @@ SUFFIXES = .bdf .pcf$(COMPRESS_SUFFIX) t
 	@rm -f $@
 	$(AM_V_GEN)$(BDFTOPCF) -t $< | $(COMPRESS) $(COMPRESS_FLAGS) > $@
 
-.bdft.bdf:
-	@rm -f $@
-	$(BDFTRUNCATE) 0x3200 < $< > $@
-
 .bdf-ISO8859-1.bdf:
 	@rm -f $@
 	$(AM_V_GEN)$(UCS2ANY) $< $(UTIL_DIR)/map-ISO8859-1 ISO8859-1
